src/redux/ducks/locs/actions.js

```diff
@@ -3,27 +3,57 @@ import LocDAO from '../../../dao/LocDAO';
 import {notify} from '../../../redux/ducks/notifier/notifier';
 import LOCNoticeModel from '../../../models/notices/LOCNoticeModel';
 import {LOCS_LOAD_START, LOCS_LOAD_SUCCESS} from './communication';
-import { createAllLOCsAction, createLOCAction, updateLOCAction, removeLOCAction } from './reducer';
+import { createAllLOCsAction, createLOCAction/*, updateLOCAction, removeLOCAction*/ } from './reducer';
+import NoticeModel from '../../../models/notices/NoticeModel';
 
 const locsLoadStartAction = () => ({type: LOCS_LOAD_START});
 const locsLoadSuccessAction = (payload) => ({type: LOCS_LOAD_SUCCESS, payload});
 
 const updateLOC = (data) => (dispatch) => {
-    AppDAO.updateLOC(data, data.account);
+    AppDAO.updateLOC(data, data.account).then( (r) => {
+        if (r === true){
+            dispatch(notify(new NoticeModel({message: 'LOC updated'})))
+        } else {
+            dispatch(notify(new NoticeModel({message: 'LOC not updated'})))
```

Comment:
You have LOCNoticeModel for LOC entity notices. Please use it here.
Also your notice should be created only when you will get new event from blockchain, because we want to restore old notices list from blockchain. Please rework this by example with settings/cbe.js:watchUpdateCBE, your watch functions in DAO should be wrapped with AbstractContractDAO:_watch (for instance see TokenContractsDAO:watch).
I'll make NoticeModel abstract and remove 'message' from it, message should be generated for each notice inside of the notice model based on it properties, because we don't save 'message' anywhere and want to translate it in future.